packages/patternfly-4/react-core/src/components/Accordion/examples/SimpleAccordion.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
```

Comment:
Try `import  * as React from 'react';`, this has fixed issues with builds in the past (and we'll just have to change it on the Typescript conversion anyways)